recipe/recipe.yaml

```
@@ -0,0 +1,73 @@
+schema_version: 1
+
+context:
+  name: fastjet
+  version: 3.5.0.0
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.io/packages/source/${{ name[0] }}/${{ name }}/fastjet-${{ version }}.tar.gz
```

```suggestion
  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/fastjet-${{ version }}.tar.gz
```

This is the linter comment, but other than that LGTM

---

It might get caught by re-render... Let's see.